src/MessagingBubble/MessagingBubble.less

```diff
@@ -10,6 +10,10 @@
   overflow-wrap: break-word;
   white-space: pre-wrap;
   word-wrap: break-word; // IE name for overflow-wrap
+  .MessagingAttachment--Container {
+    .margin--top--3xs();
+    width: 22.5rem;
+  }
```

Comment:
I don't see many other instances of nested styling, so I'm curious if this is viewed as an anti-pattern around here.

Reply:
I can provide some quick context here! We've historically tried to keep nested styling to a minimum, to avoid issues when components get restructured or DOM elements get moved around.

We have a style guide here that mentions this reasoning: https://clever.github.io/components/#/less/less-style-guide (scroll to the section on nesting)

In some repos, we actually enforce this via linting, e.g. https://github.com/Clever/family-portal/blob/e3011ee0176a192f6e41291800e91f57e0fda353/.stylelintrc.js#L28

Reply:
I should... do this more often 🙃